docs/Behaviour - Client Side.md

```diff
@@ -2,9 +2,9 @@
 
 ## Active Constraints of Sender
 
-Before making an [IS-05][IS-05] connection, NMOS Controller chooses a Sender and Receivers. Then NMOS Controller MUST `GET /constraints/supported` from the Sender, MUST collect Receiver Capabilities from the Receivers and make such intersection of them that they can be utilized for building Active Constraints satisfying all the Receivers and using Parameter Constraint URNs supported by the Sender. If the Sender supports fewer Parameter Constraint URNs than used in the Receiver Capabilities, the NMOS Controller SHOULD inform the user about it. After that NMOS Controller MUST `PUT /constraints/active` and SHOULD make the IS-05 connection if the Constraints have been applied succesfully. After breaking this connection via IS-05, NMOS Controller is RECOMMENDED to `DELETE /constraints/active`.
+Before making an [IS-05][IS-05] connection, NMOS Controller chooses a Sender and a number of  Receivers. Then NMOS Controller MUST `GET /constraints/supported` from the Sender, MUST collect Receiver Capabilities from the Receivers and make such intersection of them that they can be utilized for building Active Constraints satisfying all (default) or some (user preferences) of the Receivers and using Parameter Constraint URNs supported by the Sender. If the Sender supports fewer Parameter Constraint URNs than used in the Receiver Capabilities, the NMOS Controller SHOULD inform the user about it. After that NMOS Controller MUST `PUT /constraints/active` to the Sender and SHOULD make the IS-05 connections if the Constraints have been applied succesfully to the Sender. After breaking these connections via IS-05, NMOS Controller is RECOMMENDED to `DELETE /constraints/active` of the Sender.
 
-Constraint Sets of Active Constraints SHOULD be the largest consensus among Receivers. For example, there are Receiver A, Receiver B, Receiver C with equal Receiver Capabilities consisting of Constraint Sets 1, 2, 3, 4, 5 and Receiver D with Constraint Sets 2, 3, 4, 5 and 6 which is incompatible with the previous ones. In this case the largest consensus is Constraint Sets 2, 3, 4, 5 which are common for all the four Receivers.
+Constraint Sets of Active Constraints SHOULD represent the largest consensus among Receivers. For example, there are Receiver A, Receiver B, Receiver C with equal Receiver Capabilities consisting of Constraint Sets 1, 2, 3, 4, 5 and Receiver D with Constraint Sets 2, 3, 4, 5 and 6 which is incompatible with the previous ones. In this case the largest consensus is Constraint Sets 2, 3, 4, 5 which are common for all the four Receivers. The user may control how the largest consensus is obtained, providing preferences to the NMOS Controller among the Receivers and the Constraint Sets of each Receiver and criteria about the size of the consensus.
```

Comment:
```suggestion
Constraint Sets of Active Constraints SHOULD represent a consensus among Receivers. For example, there are Receiver A, Receiver B, Receiver C with equal Receiver Capabilities consisting of Constraint Sets 1, 2, 3, 4, 5 and Receiver D with Constraint Sets 2, 3, 4, 5 and 6 which is incompatible with the previous ones. In this case the largest consensus is Constraint Sets 2, 3, 4, 5 which are common for all the four Receivers. The user may control how the consensus is obtained, providing preferences to the NMOS Controller among the Receivers and the Constraint Sets of each Receiver and criteria about the size of the consensus.
```

I think Alain's wording is good, I'm just proposing deleting "largest" in two places to allow for the user overriding. Could instead replace with "best" or "preferred" or something...

Perhaps we should use "consensus" throughout this documentation instead of "intersection", apart from as an example of default behaviour when there is no user control? Intersection is almost too precise a term I feel.

Reply:
I think we also need to update the second part of this example two paragraphs lower down, to explain precisely what "more important" means to the user in this example.

Reply:
What about replacing "such intersection" by  "such processing" as follow: "Then NMOS Controller MUST `GET /constraints/supported` from the Sender, MUST collect Receiver Capabilities from the Receivers and make such processing of them that they can be utilized for building Active Constraints satisfying all (by default) or some (depending on user preferences) of the Receivers and using Parameter Constraint URNs supported by the Sender."

The idea is the the controller in its processing of the receiver capabilities may perform intersections based on some criteria and may also perform selections based on some other criteria. It is true that the word "intersection" may be misleading ...